doc/modules/model_evaluation.rst

```diff
@@ -6,6 +6,139 @@
+==================  ===============================================  ===================  ==============================
+**Classification**
+mean                :ref:`Brier score <brier_score_loss>`            multi-class          ``predict_proba``
+mean                :ref:`log loss <log_loss>`                       multi-class          ``predict_proba``
+mode                :ref:`zero-one loss <zero_one_loss>` :sup:`1`    multi-class          ``predict``, categorical
+**Regression**
+mean                :ref:`squared error <mean_squared_error>`        all reals             ``predict``, all reals
+mean                :ref:`Poisson deviance <mean_tweedie_deviance>`  non-negative          ``predict``, strictly positive
+mean                :ref:`Gamma deviance <mean_tweedie_deviance>`    strictly positive     ``predict``, strictly positive
+median              :ref:`absolute error <mean_absolute_error>`      all reals             ``predict``, all reals
+quantile            :ref:`pinball loss <pinball_loss>`               all reals             ``predict``, all reals
+mode                no consistent one exists                         reals
+==================  ===============================================  ===================  ==============================
+
+:sup:`1` The zero-one loss is only consistent but not strictly consistent for the mode.
+
+The zero-one loss is equivalent to one minus the accuracy score, meaning it
+gives different score values but the same ranking.
+R² gives the same ranking as squared loss.
+Furthermore, the Brier score is just a different name for the squared error
+in case of classification.
+
+**Ficticious Example:**
+Let's make the above arguments more tangible. Consider a setting of reliability
+engineering of network connections, e.g. internet or wifi. As provider of the network,
+you have access to the dataset of log entries of network connections containing network
+load over time and many interesting features. Your goal is to improve the raliability
+of the connections. In fact, you promise your customers that at least on 99% of all
+days there are no connection discontinuities larger 1 minute.
+Therefore, you are interested in a prediction of the 99% quantile (of longest
+connection interruption duration per day) in order to know in advance when to add
+more bandwidth and thereby satisfy your customers. So the *target functional* is the
+99% quantile. From the table above, you choose the pinball loss as scoring function
+(fair enough, not much choice given), for model training (e.g.
+`HistGradientBoostingRegressor(loss="quantile", quantile=0.99)`) as well as model
+evaluation (`mean_pinball_loss(..., alpha=0.9)` - we apoligize for the different
+argument names, `quantile` and `alpha`) be it in grid search for finding
+hyperparameters or in comparing to other models like
+`QuantileRegressor(quantile=0.99)`.
+
+.. topic:: References:
```

Comment:
```suggestion
.. rubric:: References
```
This is how it is done in the rest of the document; results in a slightly different formatting.